ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -683,7 +689,10 @@ def get_workflow(dict_configs, cdump='gdas'):
     deps = []
     dep_dict = {'type':'metatask', 'name':f'{cdump}post'}
     deps.append(rocoto.add_dependency(dep_dict))
-    dep_dict = {'type':'task', 'name':f'{cdump}vrfy'}
+    if do_vrfy in ['Y', 'YES']:
+        dep_dict = {'type':'task', 'name':f'{cdump}vrfy'}
+        deps.append(rocoto.add_dependency(dep_dict))
+    dep_dict = {'type':'streq', 'left':'&ARCHIVE_TO_HPSS;', 'right':'YES'}
```

Comment:
Can you explain why `YES` is hard-wired here instead of `base.get('HPSSARCH', 'NO')`
`config.base` provides a toggle flag to archive to HPSS `HPSSARCH`.

Reply:
Modified to dep_dict = {'type':'streq', 'left':'&ARCHIVE_TO_HPSS;', 'right':f'{hpssarch}'}